Source/DafnyLanguageServer.Test/Synchronization/IncludeFileTest.cs

```diff
@@ -95,9 +95,10 @@ public class IncludeFileTest : ClientBasedLanguageServerTest {
 ".TrimStart();
     var documentItem = CreateTestDocument(source, TestFilePath);
     await client.OpenDocumentAndWaitAsync(documentItem, CancellationToken);
-    var parseDiagnostics = await diagnosticsReceiver.AwaitNextDiagnosticsAsync(CancellationToken);
-    Assert.Single(parseDiagnostics);
-    Assert.Contains(parseDiagnostics, d => d.Message.Contains("cycle of includes"));
+    // Parse diagnostics are currently only sent if they contain errors
+    // var parseDiagnostics = await diagnosticsReceiver.AwaitNextDiagnosticsAsync(CancellationToken);
+    // Assert.Single(parseDiagnostics);
+    // Assert.Contains(parseDiagnostics, d => d.Message.Contains("cycle of includes"));
```

Comment:
Relates to the skipped test. I was hoping that with the comment it would be OK to keep. Might help reduce confusion if we ever change the behavior and then this test breaks, but gets fixed with the commented code.